### PR TITLE
[8.2.5] oLocker checks component permissions

### PR DESCRIPTION
**Pull request by gonzalo-martinez-imt**
Linked to issue #607

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 8.2.5
+### Features
+* **o-image**: New attribute `max-file-size`. ([d8a84c0](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/d8a84c0)) Closes [#589](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/589)
+ 
+### Bug Fixes
+* **o-list-picker, OFormDataComponent**: checking if the component has the apropiate permissions to execute the `setEnabled` method ([40089da](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/40089da)) Closes [#607](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/607)
```
Comment (daniel-grana) [inline CHANGELOG.md:6]: Typo: _appropriate_

File: projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import {
   OnInit,
   Optional,
   SimpleChange,
-  ViewChild
+  ViewChild,
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts:13]: can this trailing comma cause tslint problems?

File: projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ import { OFormControl } from '../o-form-control.class';
 import {
   DEFAULT_INPUTS_O_FORM_SERVICE_COMPONENT,
   DEFAULT_OUTPUTS_O_FORM_SERVICE_COMPONENT,
-  OFormServiceComponent
+  OFormServiceComponent,
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts:27]: can this trailing comma cause tslint problems?

**Review by daniel-grana — CHANGES_REQUESTED** [review]

File: projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts (hunk 2 above)
Comment (gonzalo-martinez-imt) [reply projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts:13]: I dont think so, but I had the typescript hero plugin misconfigured and it was adding this comma.

File: projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts (hunk 3 above)
Comment (gonzalo-martinez-imt) [reply projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts:27]: I dont think so, but I had the typescript hero plugin misconfigured and it was adding this comma.

**Review by daniel-grana — APPROVED** [review]

**Review by manuel-cid — APPROVED** [review]

**Review by pacogomezimatia — APPROVED** [review]

**Pull request merged**